src/main/java/org/benetech/servicenet/builder/ReportErrorMessageBuilder.java

```diff
@@ -29,6 +30,17 @@
         return builder.toString();
     }
 
+    public static String buildForError(String message, DataImportReport report) {
+        String prevPart;
+        if (StringUtils.isBlank(report.getErrorMessage())) {
+            prevPart = FIRST_LINE;
```

Comment:
I think blank errors shouldn't be omitted but marked as undefined

Reply:
It's not handling blank errors but rather current errorMessage for the report, which will be null for the first error